Makefile

```diff
@@ -269,6 +269,39 @@ docs-generate-k0smotron: $(CRDOC) ## Generate docs for k0smotron CRDs
 # Generate docs for all CRDs apis
 docs-generate-reference: docs-generate-bootstrap docs-generate-controlplane docs-generate-infrastructure docs-generate-k0smotron
 
+.PHONY: generate-all
```

Comment:
I think it's a bit too verbose. WDYT about just adding them as dependencies to the generate-all target?